packages/ui-admin/tsconfig_bp.json

```diff
@@ -5,9 +5,9 @@
     "baseUrl": "./",
     "paths": {
       "~/*": ["src/*"],
-      "botpress/sdk": ["../../sdk/botpress.d.ts"],
-      "botpress/shared": ["../../../../ui-shared/src/typings.d.ts"],
-      "common/*": ["../../common/*"]
+      "botpress/sdk": ["../bp/src/sdk/botpress.d.ts"],
+      "botpress/shared": ["../ui-shared/src/typings.d.ts"],
+      "common/*": ["../bp/src/common/*"]
     }
   }
 }
```

Comment:
I'm pretty sure the content of this file can be moved inside the tsconfig.json file.

Reply:
Nope! For the same reason we have some config in the .env file :P. React-scripts is a bit restrictive, so we're doing some voodoo stuff to force our way through.

![image](https://user-images.githubusercontent.com/42552874/144479723-da1841d1-6beb-4378-821b-8b1ae68c5a12.png)

This warning erases the "path" parameter in the tsconfig file. Extending is a hack, it prints the warning but removes the path in tsconfig.json, but leaves it in tsconfig_bp.json.

It's not "supported", but that's why we use react-app-rewired and that we overrides a couple of webpack settings